library/src/main/java/com/midtrans/Config.java

```diff
@@ -1,35 +1,39 @@
 package com.midtrans;
 
 import com.midtrans.proxy.ProxyConfig;
+import lombok.EqualsAndHashCode;
+
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
 
 /**
  * Midtrans configuration
  */
+@EqualsAndHashCode(callSuper = false)
 public class Config {
 
     private String SERVER_KEY;
     private String CLIENT_KEY;
     private String IRIS_MERCHANT_KEY;
```

Comment:
I just realize this, is these always been all capitals since beginning? because the function name `setSERVER_KEY` is awkward and should be regular `setServerKey`.

In node JS capitalized var name is to indicate private static vars, because NodeJS have no private static. But in java it should not be followed.

Well if it was always been capitals since beginning then we are forced to avoid change it, to avoid breaking changes. Or introduce/add new alias setter functions with the proper names.